src/makeInfinityAdder.js

```diff
@@ -5,6 +5,21 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const adder = (a, prev = sum) => {
```

Comment:
The parameter 'prev' is not necessary and can lead to confusion. The 'sum' variable already holds the state of the current sum, so you don't need to pass it as a parameter.